components/ui/web/components/content/jobs/new/newJob.tsx

```diff
@@ -0,0 +1,214 @@
+import { FC, useMemo, useState } from 'react';
+import { useRouter } from 'next/router';
+import { useQuery, ApolloError, useMutation } from '@apollo/client';
+import styled from 'styled-components';
+import { Button, Chip, IconButton, Step, StepLabel, Stepper } from '@mui/material';
+import { Close as CloseIcon } from '@mui/icons-material';
+import Swal from 'sweetalert2';
+
+import { DataVolume, Domain, Image, UserVolume } from 'src/graphql/typings';
+import { GET_DOMAINS } from 'src/graphql/domains';
+import { CREATE_JOB } from 'src/graphql/jobs';
+
+import { NewResource, NewSessionType } from 'components/content/newResource/newResource';
+import { CommandForm } from 'components/content/jobs/new/commandForm';
+
+const Styled = styled.div`
+  .header {
+    display: flex;
+    align-items: center;
+    gap: 0.5rem;
+    margin-bottom: 1.5rem;
+
+    .alert {
+      color: ${({ theme }) => theme.palette.warning.dark};
+    }
+  }
+
+  .step-buttons {
+    display: flex;
+    justify-content: space-between;
+    margin-top: 1rem;
+  }
+`;
+
+const steps = ['Select Domain & Image', 'Set Command', 'Submit'];
+export const NewJob: FC = () => {
+
+  const router = useRouter();
+
+  const [loadingSubmit, setLoadingSubmit] = useState<boolean>(false);
+  const [activeStep, setActiveStep] = useState<number>(0);
+  const [commandError, setCommandError] = useState<boolean>(false);
+
+  const { loading: loadingData, data } = useQuery(GET_DOMAINS,
+    {
+      variables: { jobs: true },
+      onError: (error: ApolloError) => {
+        if (error.message.includes('Unauthorized')) {
+          router.push('/login?callbackURL=/compute');
+        }
+      }
+    }
+  );
+
+  const [createJob] = useMutation(CREATE_JOB, {
+    onError: () => Swal.fire({
+      title: 'Unable to add docker job',
+      text: 'Please email <a href=\"mailto:sciserver-helpdesk@jhu.edu\">sciserver-helpdesk@jhu.edu</a> for more assistance.',
```

Comment:
We should be using environment variables for the mailto and other help/doc links.